### ISSUE TITLE
One Fulfilment

### ISSUE THREAD

**Issue by mohan-mandal-pattern** [allegro/allegro-api#9001]
1. How can I create a One fulfilment account based on my Live Business account? What are the requirements?
2. How can I create a One fulfilment account based on my Sandbox Business account? What are the requirements?

**Comment by MaciejFrackowiak** [comment]
1. You can find all the basic information about the service [here](https://allegro.pl/kampania/en/fulfillment). For more detailed information please choose option [Ask for a contact](https://allegro.pl/kampania/en/fulfillment#modal3).
2. All the information you received from my colleague in the thread #8976. Please, read it carefully.

**Comment by mohan-mandal-pattern** [comment]
are you sure that the Sandbox business account, has different requirements than the Live Business account?

**Comment by MaciejFrackowiak** [comment]
Yes, because it doesn't need real documents or registry numbers.
